ProjectVoyage/API Managers/APIManager.swift

```diff
@@ -0,0 +1,128 @@
+import UIKit
```

Comment:
curious: why did you write this class in swift vs. objective c?

Reply:
I shouldn't have committed this. I didn't write this myself -- this was back when I was experimenting with the speech API and found examples of API managers written in swift